src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -187,6 +196,29 @@ const CreateConceptForm = (props) => {
             </div>
           </div>
         </div>
+        {props.concept.toString().trim().toLowerCase().indexOf('set') > -1 ? (
```

Comment:
Is this testing for a concept named set?

Reply:
Yes. Improved this after a recent PR merge means this will only ever be `Set`. Made it a function as well.